src/components/RadioButton/RadioButton.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { Fragment } from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+import { action } from '../../helpers/action';
+
+import { RadioButton } from './RadioButton';
+
+storiesOf('RadioButton', module)
+	.addParameters({ jest: ['RadioButton'] })
+	.add('Radio Button', () => (
```

Comment:
Voeg meerdere radiobuttons toe in story

Reply:
Is dat niet voor de RadioButtonGroup component die nog moet gemaakt worden?

Reply:
> Voeg meerdere radiobuttons toe in story

Is dat niet voor de aparte RadioButtonGroup om te regelen? Doen we namelijk ook zo bij Checkbox en CheckboxGroup.